src/schema/schema_worker.c

```diff
@@ -128,15 +128,15 @@ __schema_layered_worker_verify(WT_SESSION_IMPL *session, const char *uri,
           ingest_ret =
             __wt_schema_worker(session, ingest_uri, file_func, name_func, cfg, open_flags));
 
-        /*
-         * FIXME-WT-15433 Add an assertion that verifying the ingest table of the leader never
-         * returns EBUSY.
-         */
+        WT_ASSERT_ALWAYS(session, ingest_ret != EBUSY,
+          "Verify(layered): %s ingest table on leader cannot be verified. "
+          "Ingest contains dirty content or open cursors, which is an invalid "
+          "state.",
+          ingest_uri);
+
         if (ingest_ret != 0)
             WT_ERR_MSG(session, ingest_ret,
-              "Verify (layered): %s ingest table verification failed. Ingest must always be empty "
-              "on leader.",
-              ingest_uri);
+              "Verify (layered): %s ingest table verification failed.", ingest_uri);
```

Comment:
I think you can do this directly.
```suggestion
WT_ERR_MSG(session, ingest_ret,
              "Verify (layered): %s ingest table verification failed.", ingest_uri);
```

Reply:
We shouldn't remove the conditional statement because that'll make `WT_ERR_MSG` always execute

Reply:
We should use the new macro from @pmacko86: `WT_ERR_MSG_CHK`! Committed 6h ago. 😆

Reply:
Happy to also keep the way it is - thanks for checking!

Reply:
Genius suggestion. Done b8dcfbed6f14735cf82ca4115fb6c07a7df5213b